src/packages.builds

```diff
@@ -36,30 +36,6 @@
 
   <ItemGroup Condition="'$(BuildAllPackages)' == 'false' AND '$(SkipNativePackageBuild)' != 'true'" >   
```

Comment:
You should remove Platforms and Targets above as well.

Reply:
Right - just removed them